sql/conseil.sql

```diff
@@ -41,6 +41,10 @@ $$;
 
 SET default_with_oids = false;
 
+CREATE TABLE tezos.processed_chain_events (
+    event_level numeric PRIMARY KEY
```

Comment:
As discussed, there should be an additional column here to state the event type.

Reply:
I'm almost done with the changes